src/socket_types/publish.rs

```diff
@@ -74,7 +74,7 @@ impl<P: Poller, M: Into<zmq::Message>> Sink for Pub<M, P> {
     }
 
     fn poll_complete(&mut self) -> Poll<(), Self::SinkError> {
-        debug!("Poll complete hit!");
+        log::debug!("Poll complete hit!");
```

Comment:
You can `use log::debug;` at the start of this file as well

Reply:
I like this more explicit, but I'll change it if you want :)